src/main/kotlin/no/nav/familie/klage/oppgave/TilordnetRessursService.kt

```diff
@@ -0,0 +1,60 @@
+package no.nav.familie.klage.oppgave
+
+import no.nav.familie.klage.infrastruktur.featuretoggle.FeatureToggleService
+import no.nav.familie.klage.infrastruktur.featuretoggle.Toggle
+import no.nav.familie.klage.infrastruktur.sikkerhet.SikkerhetContext
+import no.nav.familie.klage.oppgave.dto.SaksbehandlerDto
+import no.nav.familie.klage.oppgave.dto.SaksbehandlerRolle
+import no.nav.familie.kontrakter.felles.Tema
+import no.nav.familie.kontrakter.felles.oppgave.Oppgave
+import no.nav.familie.kontrakter.felles.oppgave.StatusEnum
+import org.springframework.stereotype.Service
+import java.util.UUID
+
+@Service
+class TilordnetRessursService(
+    private val oppgaveClient: OppgaveClient,
+    private val featureToggleService: FeatureToggleService,
+    private val behandleSakOppgaveRepository: BehandleSakOppgaveRepository,
+) {
+
+    fun hentAnsvarligSaksbehandlerForBehandlingsId(behandlingId: UUID): SaksbehandlerDto {
+        val behandleSakOppgave = behandleSakOppgaveRepository.findByBehandlingId(behandlingId)
+        val oppgave = oppgaveClient.finnOppgaveMedId(behandleSakOppgave.oppgaveId)
+        val ident = oppgaveClient.finnOppgaveMedId(behandleSakOppgave.oppgaveId).tilordnetRessurs ?: ""
```

Comment:
Trenger vi å hente oppgaven to ganger her?

Reply:
Nei, her gikk det litt fort endret til: `val tilordnetRessurs = oppgave.tilordnetRessurs ?: ""`